### PR TITLE
Support IBAN & BIC observables

### PR DESCRIPTION
**Pull request by p-l-**

### CODE REVIEW

File: core/schemas/observables/iban.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from typing import Literal
+
+from core.schemas import observable
+
+
+class Iban(observable.Observable):
```
Comment (tomchop) [inline core/schemas/observables/iban.py:6]:
```suggestion
class IBAN(observable.Observable):
```

We use uppercase for other observables made of acronyms such as CIDR. (granted, some are not compliant with this, we might change it in the future)

File: core/schemas/observables/bic.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from typing import Literal
+
+from core.schemas import observable
+
+
+class Bic(observable.Observable):
```
Comment (tomchop) [inline core/schemas/observables/bic.py:6]:
```suggestion
class BIC(observable.Observable):
```

**Review by tomchop — CHANGES_REQUESTED** [review]
Small nit regarding uppercase, otherwise LGTM!

**Comment by tomchop** [comment]
Also worth noting that if you want them to be visible from the UI you'll have to add these to observableDefinitions.json in yeti-feeds-frontend: https://github.com/yeti-platform/yeti-feeds-frontend/blob/main/src/definitions/observableDefinitions.js

**Comment by p-l-** [comment]
Hey @tomchop!

Thanks for your review! I have updated the PR (could not merge your suggestions directly since it had to be changed everywhere).

> Also worth noting that if you want them to be visible from the UI you'll have to add these to observableDefinitions.json in yeti-feeds-frontend

That's planned yes!

**Review by tomchop — APPROVED** [review]

**Comment by tomchop** [comment]
Excellent, thanks! Will also have a look at the other one (and merge it)

**Pull request merged**